### PR TITLE
Adding support for setting IS_HTTPS and DEBUG environment variables

### PR DESCRIPTION
**Pull request by andyverberne**

### CODE REVIEW

File: charts/open-notificaties/templates/configmap.yaml [hunk 1]
```diff
@@ -12,6 +12,8 @@ data:
   DB_NAME: {{ .Values.settings.database.name | toString | quote }}
   CACHE_DEFAULT: {{ .Values.settings.cache.default | toString | quote }}
   CACHE_AXES: {{ .Values.settings.cache.axes | toString | quote }}
+  DEBUG: {{ .Values.settings.debug | toString | quote }}
+  IS_HTTPS: {{ .Values.settings.isHttps | toString | quote }}
```
Comment (bartjkdp) [inline charts/open-notificaties/templates/configmap.yaml:16]: Would be great to sort this alphabetically.

File: charts/open-zaak/templates/configmap.yaml [hunk 2]
```diff
@@ -12,6 +12,8 @@ data:
   DB_NAME: {{ .Values.settings.database.name | toString | quote }}
   CACHE_DEFAULT: {{ .Values.settings.cache.default | toString | quote }}
   CACHE_AXES: {{ .Values.settings.cache.axes | toString | quote }}
+  DEBUG: {{ .Values.settings.debug | toString | quote }}
+  IS_HTTPS: {{ .Values.settings.isHttps | toString | quote }}
```
Comment (bartjkdp) [inline charts/open-zaak/templates/configmap.yaml:16]: Would be great to sort this alphabetically.

File: charts/open-zaak/values.yaml [hunk 3]
```diff
@@ -123,6 +123,10 @@ settings:
     enabled: false
     mapperFile: ""
 
+  isHttps: "True"
```
Comment (bartjkdp) [inline charts/open-zaak/values.yaml:126]:
```suggestion
  isHttps: true
```

File: charts/open-zaak/values.yaml [hunk 4]
```diff
@@ -123,6 +123,10 @@ settings:
...
+
+  debug: "False"
```
Comment (bartjkdp) [inline charts/open-zaak/values.yaml:128]:
```suggestion
  debug: false
```

File: charts/open-notificaties/values.yaml [hunk 5]
```diff
@@ -122,6 +122,10 @@ settings:
   celeryBrokerUrl: amqp://guest:guest@open-notificaties-rabbitmq:5672//
   celeryResultBackend: amqp://guest:guest@open-notificaties-rabbitmq:5672//
 
+  isHttps: "True"
```
Comment (bartjkdp) [inline charts/open-notificaties/values.yaml:125]:
```suggestion
  isHttps: true
```

File: charts/open-notificaties/values.yaml [hunk 6]
```diff
@@ -122,6 +122,10 @@ settings:
...
+
+  debug: "False"
```
Comment (bartjkdp) [inline charts/open-notificaties/values.yaml:127]:
```suggestion
  debug: false
```

File: charts/open-notificaties/templates/configmap.yaml [hunk 7]
```diff
@@ -12,6 +12,8 @@ data:
   DB_NAME: {{ .Values.settings.database.name | toString | quote }}
   CACHE_DEFAULT: {{ .Values.settings.cache.default | toString | quote }}
   CACHE_AXES: {{ .Values.settings.cache.axes | toString | quote }}
+  DEBUG: {{ .Values.settings.debug | toString | quote }}
```
Comment (bartjkdp) [inline charts/open-notificaties/templates/configmap.yaml:15]: I would use the string values `True` and `False` as described in the manual like this:

```suggestion
  DEBUG: {{ if .Values.settings.debug }}"True"{{ else }}"False"{{ end }}
```

**Comment by bartjkdp** [comment]
Thanks for the contribution @andyverberne, great add an option to make `DEBUG` and `IS_HTTPS` configurable. I added some minor remarks. After that, this one is ready to merge 👍

File: charts/open-notificaties/templates/configmap.yaml (hunk 1 above)
Comment (andyverberne) [reply charts/open-notificaties/templates/configmap.yaml:16]: I sorted the configmap.yaml and secret.yaml of the open-notificaties and the open-zaak chart.
Notice that I kept some entries in 'logical' order. i,e,:
```
  DB_HOST: {{ .Values.settings.database.host | toString | quote }}
  DB_PORT: {{ .Values.settings.database.port | toString | quote }}
  DB_USER: {{ .Values.settings.database.username | toString | quote }}
  DB_NAME: {{ .Values.settings.database.name | toString | quote }}
```
This seems a logical order to me.

**Comment by bartjkdp** [comment]
Looking great 👍  thanks for your contribution @andyverberne!

**Pull request merged**